core/vm/instructions.go

```diff
@@ -592,13 +591,8 @@ func opCreate(pc *uint64, interpreter *EVMInterpreter, scope *ScopeContext) ([]b
 	stackvalue := size
 
 	scope.Contract.UseGas(gas)
-	//TODO: use uint256.Int instead of converting with toBig()
-	var bigVal = big0
-	if !value.IsZero() {
-		bigVal = value.ToBig()
-	}
 
-	res, addr, returnGas, suberr := interpreter.evm.Create(scope.Contract, input, gas, bigVal)
+	res, addr, returnGas, suberr := interpreter.evm.Create(scope.Contract, input, gas, new(uint256.Int).Set(&value))
```

Comment:
Yeah, should be consistent, you are right. However, I think it would be better to just use `&value`. The `value` is a `uint256.Int` (not pointer), just popped off the stack. So nobody else can be holding a reference to it. And it's not used after this line.